core/color.h

```diff
@@ -7,65 +7,67 @@
 
 #include <QColor>
 
+#define MAKEQCOLOR(r,g,b,a) QColor::fromRgbF((float)(r), (float)(b), (float)(g), (float)(a))
```

Comment:
I think you inverted blue and green here...? That must look funky?

Personally, I would find an inline function easier to read, such as
```
inline QColor makeColor(double r, double g, double b, double a = 1.0)
{
    return QColor::fromRgbF((float)r, (float)g, float(b), (float)a);
}
```
Thus, you could also remove the default `1` argument in most cases. But potato, patato.

Reply:
that was very clever of me to mess with the color components...
And I'm happy to do an inline function instead...
And to have it use the `float` cast only in Qt6

Reply:
fixed based on your recommendation